modules/calypsoify/build-sass.sh

```diff
@@ -0,0 +1,4 @@
+#!/bin/bash
+
+sass style.scss | perl -pe 's/^    /\t\t/g' | perl -pe 's/^  /\t/g' > style.css
```

Comment:
Let's add these to our existing build process

Reply:
Sure.  For the moment this was faster and easier to just do in place.

Reply:
Done.